draft-ietf-jsonpath-base.md

```diff
@@ -1173,6 +1173,11 @@ They have the normal semantics of Boolean algebra and obey its laws
 (see, for example, {{BOOLEAN-LAWS}}).
 Parentheses MAY be used within `logical-expr` for grouping.
 
+It is not required that `logical-expr` consist of
```

Comment:
We now have removed all examples that actually show what is discussed here?
(We know what it is, but readers may not.)

Reply:
Yes, I was a little concerned about that too, so I've added a specific example near the top of the filter example table.